### DISCUSSION TITLE
Display Graph Legend along with graph using neovis.js

### DISCUSSION THREAD

**Discussion by AntonyNitheesh** [neo4j-contrib/neovis.js#286] [Ideas]
Hi all,
I am thinking of can we display the graph legend along with our graph using noevis.js. 
So it will be helpful for users to identify which color node belongs to which graph legend.

**Comment by subashw** [comment]
If you use the underlying library vis.js which neovis wraps you can do it.

**Comment by subashw** [comment]
https://rdrr.io/cran/visNetwork/man/visLegend.html

**Comment by Niveydhithaa** [comment]
Hi, Ive been trying to add a tooltip on my neovis graph/ The tool tip can display the node name, proeprties if any..  

But all my tries using the function 
`
function: {
    'title': (node) => NeoVis.objectToTitleHtml(node, [
        "name",
        "label"
    ])
    // 'title': (node) => NeoVis.objectToTitleHtml(node)
}
`

Can someone help me out please? Thanks
